### PR TITLE
Renew Membership Flow

### PR DESCRIPTION
**Pull request by chriskrogh**
## What's changed
Completed the renew membership flow

## UI (if UI has changed)
Desktop
Screen-Recording-2020-08-05-at-1

Mobile
Screen-Recording-2020-08-05-at-1 (1)

Looks good on tablet too😂 I checked

## Notes
I'll work on backend integration sometime. 
But we should work on edge cases.
E.g. if a user tries to sign up but their email / student id is already in use.
same thing with membership renewal

Need to start handling those from UI side

### CODE REVIEW

File: client/src/components/VerifyInfo/index.stories.tsx [hunk 1]
```diff
@@ -8,10 +8,10 @@ export default {
 
 const baseProps = {};
 
-export const Base = () => <VerifyInfo {...baseProps} />;
+export const Base = () => <VerifyInfo {...baseProps} onVerify={() => {}} />;
```
Comment (jtaaa) [inline client/src/components/VerifyInfo/index.stories.tsx:11]: We can use `action('onVerify')` for the `onVerify` prop of these stories, imported from `@storybook/addon-actions` I think.

That way we'll get a little action notification in the addons panel 🚀

File: client/src/pages/Membership/VerifyInfo/index.tsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+const styles = (theme: Theme) =>
+  createStyles({
+    container: {
+      display: 'flex',
+      flexDirection: 'column',
+      alignItems: 'center',
+      width: `calc(100% - ${2 * theme.spacing(3)}px)`,
+      padding: `0 ${theme.spacing(3)}px`,
+    },
+    contentContainer: {},
```
Comment (jtaaa) [inline client/src/pages/Membership/VerifyInfo/index.tsx:17]: this fellow isn't needed?

File: client/src/components/VerifyInfo/index.tsx [hunk 3]
```diff
@@ -19,7 +19,11 @@ const styles = (theme: Theme) =>
     },
   });
 
-function VerifyInfo({ classes }: WithStyles<typeof styles>) {
+type Props = WithStyles<typeof styles> & {
+  onVerify: () => void;
```
Comment (jtaaa) [inline client/src/components/VerifyInfo/index.tsx:23]: Will `onVerify` need to be async to query the backend?

File: client/src/pages/Membership/VerifyInfo/index.tsx (hunk 2 above)
Comment (chriskrogh) [reply client/src/pages/Membership/VerifyInfo/index.tsx:17]: No he is not! Thanks for the catch!

File: client/src/components/VerifyInfo/index.tsx (hunk 3 above)
Comment (chriskrogh) [reply client/src/components/VerifyInfo/index.tsx:23]: Yessir. Will update in next PR

**Pull request merged**